src/main/java/teammates/ui/webapi/CreateAccountRequestAction.java

```diff
@@ -31,8 +31,7 @@ public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOpera
         } catch (InvalidParametersException ipe) {
             throw new InvalidHttpRequestBodyException(ipe);
         } catch (EntityAlreadyExistsException eaee) {
-            // Use existing account request
-            accountRequestAttributes = logic.getAccountRequest(instructorEmail, instructorInstitution);
+            throw new InvalidOperationException("Account request already exists.", eaee);
```

Comment:
This should be thrown only if the account request has been registered. Reusing existing unregistered one is harmless.